client/src/app/management/components/meeting-edit/meeting-edit.component.ts

```diff
@@ -198,26 +200,36 @@ export class MeetingEditComponent extends BaseModelContextComponent implements O
         const currentDate = new Date();
         currentDate.setHours(0, 0, 0, 0);
 
-        const rawForm: { [key: string]: any } = {
-            name: ['', Validators.required],
-            // server bug
-            // set_as_template: [false],
-            description: [''],
-            location: [''],
-            start_time: [currentDate],
-            end_time: [currentDate],
-            user_ids: [[]],
-            admin_ids: [[], Validators.minLength(1)],
-            organization_tag_ids: [[]]
-        };
+        const action = this.isCreateView ? MeetingAction.CREATE : MeetingAction.UPDATE;
+        const additionalFormControls = { admin_ids: [[], Validators.minLength(1)] };
+        const rawForm = this.formgroupGenerator.generateFormGroup(action, additionalFormControls, {
+            start_time: { defaultValue: currentDate },
+            end_time: { defaultValue: currentDate }
+        });
```

Comment:
Then we can use those descriptions like this.
What do you mean @tsiegleauq and @FinnStutzenstein ?